espnet/nets/beam_search.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, scorers: Dict[str, ScorerInterface], weights: Dict[str, float
         super().__init__()
         # set scorers
         self.weights = weights
+        self.scorers = dict()
```

Comment:
This `scorers` is used for only removing `chain` method?

Reply:
It is also used in `BatchScorer.batchfy` and it looks cleaner for me.

Reply:
I see.